### PR TITLE
Added whl2conda recipe

### PR DESCRIPTION
**Pull request by analog-cbarber**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by analog-cbarber** [comment]
@conda-forge/help-python, ready for review

**Comment by analog-cbarber** [comment]
@conda-forge/help-python, ready for review!

**Comment by analog-cbarber** [comment]
@conda-forge-admin, please ping conda-forge/python ready for review

I can confirm that I am the maintainer of the GitHub repo and am happy to be the feedstock maintainer for this Conda package.

File: recipes/whl2conda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "whl2conda" %}
+{% set version = "23.8.4" %}
+
+package:
+  name: whl2conda
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/w/whl2conda/whl2conda-{{ version }}-py3-none-any.whl
```
Comment (ocefpaf) [inline recipes/whl2conda/meta.yaml:9]: The irony is not lost on me but can we build it from source instead of repacking the wheel?

Comment (analog-cbarber) [reply recipes/whl2conda/meta.yaml:9]: I am sure it is possible to build from source, but does it serve any purpose? I am not currently publishing sdists. There is already a wheel on pypi and so building from source is just busy work.

Really, I would like to just bypass conda build entirely and use this tool to build itself. I could do that if I were to publish to my own anaconda channel, but I don't know if that is even possible with the conda-forge infrastructure.

Comment (analog-cbarber) [reply recipes/whl2conda/meta.yaml:9]: There are other feedstocks that build from wheels, so I assumed it was kosher. E.g.

https://github.com/conda-forge/keras-feedstock/blob/main/recipe/meta.yaml

Comment (ocefpaf) [reply recipes/whl2conda/meta.yaml:9]: Nope. It is a last resort measure and we really want to avoid it.

Comment (analog-cbarber) [reply recipes/whl2conda/meta.yaml]: Ok. Switched to build from sdist.

**Pull request merged**